<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rentacar</class>
 <widget class="QWidget" name="Rentacar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <spacer name="verticalSpacer_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Assign a Car to Customer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="labelAssignCarId_2">
               <property name="styleSheet">
                <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Car Id: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelAssignCustomerId_2">
               <property name="styleSheet">
                <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Customer Id: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelRentStarts_2">
               <property name="styleSheet">
                <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Rent Starts:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelRentEnds_2">
               <property name="styleSheet">
                <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Rent Ends: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEditAssignCarId_2">
               <property name="styleSheet">
                <string notr="true">QLineEdit { background-color: transparent; }</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditAssignCustomerId_2">
               <property name="styleSheet">
                <string notr="true">QLineEdit { background-color: transparent; }</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEditRentStarts_2">
               <property name="styleSheet">
                <string notr="true">QLineEdit { background-color: transparent; }</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEditRentEnds_2">
               <property name="styleSheet">
                <string notr="true">QLineEdit { background-color: transparent; }</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAssigningCarToCustomer">
           <property name="maximumSize">
            <size>
             <width>316</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 

}


</string>
           </property>
           <property name="text">
            <string>Assign Car to Customer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labelRentalAgreement">
           <property name="styleSheet">
            <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Rental Agreements&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tableViewBooking">
           <property name="styleSheet">
            <string notr="true">QTableView {
    background-color: transparent;
    gridline-color: #b5b5b5; 
}

QTableView::item {
    border: 1px solid #b5b5b5; 
}

QHeaderView::section {
    background-color: transparent;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelResultBooking">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Unassign&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Rent Id: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRentId_2">
             <property name="styleSheet">
              <string notr="true">QLineEdit { background-color: transparent; }</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonUnassigningAgreement">
           <property name="styleSheet">
            <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
           </property>
           <property name="text">
            <string>Unassign Agreement</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QPushButton" name="pushButtonBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_13">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableView" name="tableViewSearchCar">
           <property name="styleSheet">
            <string notr="true">QTableView {
    background-color: transparent;
    gridline-color: #b5b5b5; 
}

QTableView::item {
    border: 1px solid #b5b5b5; 
}

QHeaderView::section {
    background-color: transparent;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelCarTable">
             <property name="styleSheet">
              <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>Cars</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRefreshCarTable">
             <property name="styleSheet">
              <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 

}


</string>
             </property>
             <property name="text">
              <string>Refresh Table</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labelSearchCar">
           <property name="styleSheet">
            <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);

</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Search Car&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelRegistrationNumber_2">
             <property name="styleSheet">
              <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Registration Number: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSearchRegistrationNumber_2">
             <property name="styleSheet">
              <string notr="true">QLineEdit { background-color: transparent; }</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearchCar_2">
             <property name="styleSheet">
              <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
             </property>
             <property name="text">
              <string>Search Car</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableView" name="tableViewSearchCustomer">
           <property name="styleSheet">
            <string notr="true">QTableView {
    background-color: transparent;
    gridline-color: #b5b5b5; 
}

QTableView::item {
    border: 1px solid #b5b5b5; 
}

QHeaderView::section {
    background-color: transparent;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelSearchCustomerTable">
             <property name="styleSheet">
              <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>Customers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRefreshCustomerTable">
             <property name="styleSheet">
              <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 

}


</string>
             </property>
             <property name="text">
              <string>Refresh Table</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labelSearchCustomer">
           <property name="styleSheet">
            <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Search Customer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="labelSearchCustomerName_2">
             <property name="styleSheet">
              <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Customer Name: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSearchCustomerName_2">
             <property name="styleSheet">
              <string notr="true">QLineEdit { background-color: transparent; }</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearchCustomer_2">
             <property name="styleSheet">
              <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
             </property>
             <property name="text">
              <string>Search Customer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_14">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
